api/notd/token_manager.py

```diff
@@ -396,6 +402,7 @@ async def update_activity_for_all_collections(self) -> None:
         logging.info(f'Scheduling processing for {len(registryDatePairs)} registryDatePairs')
         messages = [UpdateActivityForCollectionMessageContent(address=address, startDate=startDate).to_message() for (address, startDate) in registryDatePairs]
         await self.tokenQueue.send_messages(messages=messages)
+        await self.saver.update_latest_update(latestUpdateId=latestUpdate.latestUpdateId, date=date_util.datetime_from_now())
```

Comment:
date should not equal now. you should create a variable like `startDate` at the top of this funtion and set this to that